test/test_torch.py

```diff
@@ -885,6 +885,11 @@ def test_logsumexp(self):
         expected = logsumexp(a.numpy(), 1)
         self.assertEqual(expected.shape, actual.shape)
         self.assertTrue(np.allclose(expected, actual.numpy()))
+        # check that out is actually inplace
+        b = torch.zeros(5, 2)
+        c = b[:, 0]
+        torch.logsumexp(a, 1, out=c)
+        self.assertTrue(np.allclose(expected, b[:, 0].numpy()))
```

Comment:
This might be better written as `self.assertEqual(c, b[:,0])` to match the rest of the pytorch tests.